src/bp/core/repositories/sessions.ts

```diff
@@ -137,12 +167,33 @@ export class KnexSessionRepository implements SessionRepository {
     }
 
     await req
+
+    if (this.cache.has(session.id)) {
+      this.cache.del(session.id)
+    } else if (this.ownershipCache.has(session.id)) {
+      this.changeSessionOwnership(session.id, undefined)
+    }
```

Comment:
This part of the code makes the whole caching useless for now. I'll replace it with an update batch.